app/src/main/res/layout/activity_mission_in_progress.xml

```diff
@@ -91,6 +91,23 @@
         app:layout_constraintTop_toTopOf="parent"
         app:layout_constraintVertical_bias="0.974" />
 
+    <TextView
+        android:id="@+id/warningBadWeather"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:lineSpacingExtra="14sp"
+        android:text="@string/warning_bad_weather"
+        android:textAlignment="center"
+        android:textColor="#FF0000"
```

Comment:
It would have been better to add red to values/colors.xml and then just replace this line by
android:textColor="@colors/red"